site/content/docs/5.3/migration.md

```diff
@@ -107,6 +107,8 @@ Learn more by reading the new [color modes documentation]({{< docsref "/customiz
 
 - `.form-check` and `.form-switch` components are now built with CSS variables for setting the `background-image`. The usage here differs from other components in that the various focus, active, etc states for each component aren't set on the base class. Instead, the states override one variable (e.g., `--bs-form-switch-bg`).
 
+- Form text examples have been modified to add some precisions; form text should be explicitly associated with the form control it relates to using the `aria-labelledby` (for mandatory information such as data format) or `aria-describedby` (for complementary information) attribute. Please apply this modification in your websites if needed.
```

Comment:
Do we need to add a `warning` badge here since there maybe something to check on user side ?

Reply:
Yep OK I don't mind adding it. See https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/pull/1828/commits/73c3507a10bab8ab2caa3b6f32f3c70a60c3bf86